### ISSUE TITLE
bug: After the 1.5Rpm package was installed in centos7.6(ARM64), an error was occurred

### ISSUE THREAD

**Issue by idbeta** [apache/apisix#2161]
### Issue description
After the 1.5Rpm package was installed in centos7.6(ARM64), an error was occurred
[root@ecs-9148 ~]# apisix start
nginx: [error] init_by_lua error: error loading module 'lfs' from file '/usr/local/apisix//deps/lib/lua/5.1/lfs.so':
	/usr/local/apisix//deps/lib/lua/5.1/lfs.so: cannot open shared object file: No such file or directory
stack traceback:
	[C]: at 0xffffa561c9b8
	[C]: in function 'require'
	/usr/local/apisix/apisix/core/request.lua:18: in main chunk
	[C]: in function 'require'
	/usr/local/apisix/apisix/core.lua:30: in main chunk
	[C]: in function 'require'
	/usr/local/apisix/apisix/init.lua:18: in main chunk
	[C]: in function 'require'
	init_by_lua:3: in main chunk

### Environment

* apisix version (cmd: `apisix version`): 1.5
* OS: centos7.6(ARM64)

**Comment by Fuchange** [comment]
when I build apisix-2.1 rpm package and run at openEuler 20.09 on ARM , I got the same erro.

This is because the file is builded on x86_ 64 system .(The path is different because I use lua 5.3 .)

```bash
# file /usr/local/apisix//deps/lib/lua/5.3/lfs.so
/usr/local/apisix//deps/lib/lua/5.3/lfs.so: ELF 64-bit LSB shared object, x86-64, version 1 (SYSV), dynamically linked, BuildID[sha1]=d4d35b43c6d757f2ff09619b75eae20bdbea78bd, not stripped
```
I think it's a bug for [build-tools/run.sh](https://github.com/api7/build-tools/blob/master/run.sh) , 

you can fix it temporary by run cmd: `yum install lua-filesystem` to overwrite it.

**Comment by membphis** [comment]
![image](https://user-images.githubusercontent.com/6814606/101501547-d70b5600-39aa-11eb-900e-f9d82d78984f.png)

**Comment by spacewander** [comment]
Solved since 2.2
